setup.py

```diff
@@ -28,13 +28,29 @@
 
 clang = False
 icc = False
+gcc = True
+
 try:
-    if os.environ['CC'] == "clang":
-        clang = True
+    if os.environ['CC'] == "gcc":
+        gcc = True
```

Comment:
```suggestion
    if "gcc" in os.environ['CC']:
        gcc = True
```



I would check for `gcc` in `CC`, etc.. Not only for exact matches

```
$ dpkg -S bin/gcc-[0-9]
gcc-11: /usr/bin/gcc-11
gcc-10: /usr/bin/gcc-10
gcc-13: /usr/bin/gcc-13
gcc-12: /usr/bin/gcc-12
$ dpkg -S bin/*-gcc-[0-9]
gcc-12-i686-linux-gnu: /usr/bin/i686-linux-gnu-gcc-12
gcc-13: /usr/bin/x86_64-linux-gnu-gcc-13
gcc-12: /usr/bin/x86_64-linux-gnu-gcc-12
gcc-11: /usr/bin/x86_64-linux-gnu-gcc-11
gcc-10: /usr/bin/x86_64-linux-gnu-gcc-10
gcc-9: /usr/bin/x86_64-linux-gnu-gcc-9
```

Reply:
Indeed, I do not want to match on x86_64-linux-gnu-gcc-13.
Only plain gcc.
Will add a comment to clarify that.